vsphere/README.md

```diff
@@ -53,9 +53,10 @@ The following scenarios are possible when using the vSphere integration:
-| `ssl_capath`              | No       | Set to the absolute file path of a directory containing CA certificates in PEM format.                                                                                                                                                                                                                                                                           |
-| `host_include_only_regex` | No       | Use a regex like this if you want the check to only fetch metrics for these ESXi hosts and the VMs running on it.                                                                                                                                                                                                                                                |
-| `vm_include_only_regex`   | No       | Use a regex to include only the VMs that are matching this pattern.                                                                                                                                                                                                                                                                                              |
+| `ssl_capath`              | No       | Set to the absolute file path of a directory containing CA certificates in PEM format.                                                                                                                                                                                                                                                                            |
+| `resource_filters`        | No       | Set parameters (`resource`, `property`, `type` `patterns`) to apply a filter to a VM, host, datastore, data center or cluster. **Requires Datadog Agent v6, v7.18+.**                                                                                                                                                                                         |
+| `host_include_only_regex` | No       | Use a regex like this if you want the check to only fetch metrics for these ESXi hosts and the VMs running on it. **Note**: This is a legacy option. Use `resource_filters` for Datadog Agent v6, v7.18.                                                                                                                                                                                                                                                        |
+| `vm_include_only_regex`   | No       | Use a regex to include only the VMs that are matching this pattern. **Note**: This is a legacy option. Use `resource_filters` for Datadog Agent v6, v7.18.                                                                                                                                                                                                                                                                                                             |
```

Comment:
Most parameters in this table don't exist anymore and should be removed:
- vm_include_only_regex
- host_include_only_regex
- include_only_marked
- all_metrics
- event_config

Other new configs can be found here: https://github.com/DataDog/integrations-core/blob/master/vsphere/datadog_checks/vsphere/data/conf.yaml.example

Reply:
Thanks Florian! Going to remove the ones that don't exist anymore in this PR and create a card so we can document the new ones. 👍